exporter/util.go

```diff
@@ -175,6 +175,9 @@ func FetchJson(ctx context.Context, logger log.Logger, endpoint string, c config
 	if req.Header.Get("Accept") == "" {
 		req.Header.Add("Accept", "application/json")
 	}
+    if req.Header.Get("Accept") == "none"{
```

Comment:
Modify your config file to use the correct value of `Accept` header instead of modifying the code.

Reply:
understand, thank you !